search/tasks_test.py

```diff
@@ -40,3 +56,180 @@ def test_increment_document_integer_field_task(mocked_api):
     increment_document_integer_field(*indexing_api_args)
     assert mocked_api.increment_document_integer_field.call_count == 1
     assert mocked_api.increment_document_integer_field.call_args[0] == indexing_api_args
+
+
+def test_wrap_retry_exception():
+    """wrap_retry_exception should raise RetryException when other exceptions are raised"""
+    with wrap_retry_exception(KeyError):
+        # Should not raise an exception
+        pass
```

Comment:
Can you explain what this is doing? Is that docstring accurate?

Reply:
That's a little confusing, it's meant to show that when an exception is not raised inside the context, it's not raised outside the context either. I'll see if I can modify to explain further